bigbluebutton-html5/imports/startup/server/index.js

```diff
@@ -43,6 +44,28 @@ Meteor.startup(() => {
 
   setMinBrowserVersions();
 
+  Meteor.setInterval(() => {
+    const currentTime = Date.now();
+    Logger.info('Checking for inactive users');
+    const users = Users.find({
+      connectionStatus: 'online',
+      lastPing: {
+        $lt: (currentTime - INTERVAL_TIME), // get user who has not pinged in the last 10 seconds
+      },
+    }).fetch();
+    if (!users.length) return Logger.info('No inactive users');
+    Logger.info('Removing inactive users');
+    users.forEach((user) => {
+      const loginTimeSec = (currentTime - user.loginTime) / 1000;
+      const lastPingSec = (currentTime - user.lastPing) / 1000;
+      user.requesterUserId = user.userId;
+      return loginTimeSec >= 10
+      && lastPingSec >= 10
```

Comment:
Would it not be better to use the INTERVAL_TIME constant here rather than a literal value?

Reply:
`(currentTime - INTERVAL_TIME) <= user.lastPing`?

Reply:
I think you'd just want to not convert the time to seconds and use the const instead of the literal.
```
const lastPingSec = currentTime - user.lastPing;
lastPingSec >= INTERVAL_TIME;
```